warehouse/query-core/src/main/java/datawave/query/tables/keyword/KeywordUUIDChainStrategy.java

```diff
@@ -13,17 +16,22 @@
 import datawave.microservice.query.Query;
 import datawave.query.exceptions.DatawaveFatalQueryException;
 import datawave.query.tables.chained.strategy.FullChainStrategy;
+import datawave.query.tables.keyword.extractor.TagCloudInputExtractor;
 
 /**
  * Strategy for chaining UUID lookup and keyword extraction queries together, delegates to StatefulKeywordUUIDChainStrategy to handle individual batches of
  * lookups.
  */
 public class KeywordUUIDChainStrategy extends FullChainStrategy<Entry<Key,Value>,Entry<Key,Value>> {
+    public static final String CATEGORY_PARAMETER = "tag.cloud.category";
+    private static final String KEYWORD_CATEGORY = "keyword";
+    private static final String DEFAULT_CATEGORY = KEYWORD_CATEGORY;
```

Comment:
We'll end up splitting whatever we put in DEFAULT_CATEGORY, so maybe make it plural?
```suggestion
    private static final String DEFAULT_CATEGORIES = KEYWORD_CATEGORY;
```